jekyll/_cci2/env-vars.md

```diff
@@ -133,26 +170,29 @@ allowing you to use interpolation and share environment variables across `run` s
 The `$BASH_ENV` workaround only works with `bash`. Other shells probably won't work.
 
 ### Alpine Linux
+{:.no_toc}
 
 An image that's based on [Alpine Linux](https://alpinelinux.org/) (like [docker](https://hub.docker.com/_/docker)), uses the `ash` shell. 
 
 To use environment variables with `ash`, just add these 2 parameters to your job.
 
 ```yaml
+version: 2.1
+
 jobs:
   build:    
-
     shell: /bin/sh -leo pipefail
     environment:
       - BASH_ENV: /etc/profile
 ```
 
 ## Setting an Environment Variable in a Shell Command
 
-While CircleCI does not support interpolation when setting environment variables, it is possible to set variables for the current shell by [using `BASH_ENV`](#using-bash_env-to-set-environment-variables). This is useful for both modifying your `PATH` and setting environment variables that reference other variables.
+While CircleCI does not support interpolation when setting environment variables, it is possible to set variables for the current shell by [using `BASH_ENV`](#using-parameters-and-bashenv). This is useful for both modifying your `PATH` and setting environment variables that reference other variables.
```

Comment:
```suggestion
While CircleCI does not support interpolation when setting environment variables, it is possible to set variables for the current shell by [using `BASH_ENV`](#using-parameters-and-bash_env). This is useful for both modifying your `PATH` and setting environment variables that reference other variables.
```

Reply:
ohhhh right! Thanks so much for this @teesloane it's been doing my head in. Will see if it works :-)